src/lib.rs

```diff
@@ -142,6 +142,32 @@ pub async fn select_brd(write_sink: &mut Arc<Mutex<impl Write>>, name: &str) ->
     send_msg(write_sink, Sk8brdMsgs::MsgSelectBoard, name.as_bytes()).await
 }
 
+pub async fn send_break(write_sink: &mut Arc<Mutex<impl Write>>) -> anyhow::Result<()> {
+    send_ack(write_sink, Sk8brdMsgs::MsgSendBreak).await
+}
+
+pub async fn send_pwr_ctrl(
+    write_sink: &mut Arc<Mutex<impl Write>>,
+    en: bool,
+) -> anyhow::Result<()> {
```

Comment:
Split into two functions?

Reply:
i think i may even inline them at this point..